lib/streamlit/cli.py

```diff
@@ -207,6 +209,10 @@ def main_run(target, args=None, **kwargs):
 
     _apply_config_options_from_cli(kwargs)
 
+    file_extension = target.split(".")[-1]
```

Comment:
Should we validate this first? what happend if have not any "." or if have more than 1?

Reply:
Should we use `os.path` here?
```
import os
_, extension = os.path.splitext(filename)
```

Reply:
i didn't have that one, thanks !